src/main/scala/sbtghactions/WorkflowStep.scala

```diff
@@ -28,22 +28,35 @@ sealed trait WorkflowStep extends Product with Serializable {
   def timeout: Option[FiniteDuration]
 }
 
+object Action {
+  val checkout = UseRef.Public("actions", "checkout", "v5")
+  val setupGraalvm = UseRef.Public("graalvm", "setup-graalvm", "v1")
+  val setupJava = UseRef.Public("actions", "setup-java", "v5")
+  val setupSbt = UseRef.Public("sbt", "setup-sbt", "v1")
+  val tmate = UseRef.Public("mxschmitt", "action-tmate", "v2")
+  val upload = UseRef.Public("actions", "upload-artifact", "v4")
+  val download = UseRef.Public("actions", "download-artifact", "v5")
+  val configurePagefile = UseRef.Public("al-cheb", "configure-pagefile-action", "v1.4")
+
+  val all = Seq(checkout, setupGraalvm, setupJava, setupSbt, tmate, upload, download, configurePagefile)
+}
```

Comment:
For my understanding, is there a significance to moving actions here? The change relaxes the validation but codegen logic is same right?